charts/incubator/minecraft-bungeecord/questions.yaml

```diff
@@ -119,61 +103,76 @@ questions:
               - value: "CUSTOM"
                 description: "Custom"
         - variable: BUNGEE_JAR_URL
-          label: "BUNGEE JAR URL"
+          label: "Bunge Jar URL"
           description: "Used to download a comma separated list of *.jar urls to the plugins folder."
           schema:
             type: string
             default: ""
             required: [[TYPE, "=", "CUSTOM"]]
             show_if: [[TYPE, "=", "CUSTOM"]]
         - variable: PLUGINS
-          label: "PLUGINS"
-          description: "Used to download a comma seperated list of *.jar urls to the plugins folder."
+          label: "Plugins"
+          description: "Used to download a comma separated list of *.jar urls to the plugins folder."
           schema:
             type: string
             default: ""
         - variable: SPIGET_PLUGINS
-          label: "SPIGET PLUGINS"
+          label: "Spiget Plugins"
           description: "The SPIGET_PLUGINS variable can be set with a comma-separated list of SpigotMC resource IDs."
           schema:
             type: string
             default: ""
         - variable: ENABLE_RCON
-          label: "ENABLE RCON"
+          label: "Enable RCON"
           schema:
             type: boolean
             default: true
-        - variable: RCON_PORT
-          label: "RCON PORT"
-          description: "Define the port for rcon"
-          schema:
-            type: string
-            default: "25575"
-            show_if: [[TYPE, "=", "ENABLE_RCON"]]
         - variable: REPLACE_ENV_VARIABLES
-          label: "REPLACE ENV VARIABLES"
-          description: "replace variables that match your defined environment variables in all the files in /server."
+          label: "Replace Env Variables"
+          description: "Replace variables that match your defined environment variables in all the files in /server."
           schema:
             type: boolean
             default: false
         - variable: ONLINE_MODE
-          label: "Enable/Disable Online Mode"
+          label: "Enable Online Mode"
+          description: "MUST set to false which is required by bungeecord"
           schema:
             type: boolean
             default: false
-        - variable: advanced
+        - variable: advancedenvironment
```

Comment:
This was not really wrong, I just like to avoid using "common" names to avoid conflicts.

Reply:
this was copied from minecraft-java